src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```diff
@@ -87,6 +87,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     </ItemGroup>
 
     <ItemGroup>
+      <LinkerArg Condition="$(EnableSourceLink) == 'true'" Include="/SOURCELINK:$(NativeIntermediateOutputPath)%(ManagedBinary.Filename).sourcelink" />
```

Comment:
`EnableSourceLink` is from the SDK, right?

Reply:
Yup, if there's a property that controls this behavior for non-AOT cases, we tend to just reuse that. SDK already defined this.